File: app/src/main/java/com/expansion/lg/kimaru/training/objs/SessionTopic.java
package com.expansion.lg.kimaru.training.objs;

/**
 * Created by kimaru on 2/21/18.
 */

public class SessionTopic {

    String name, country, metaData, comment, dateAdded;
    boolean archived;
    Integer id, addedBy;

    public SessionTopic() {
    }

    public SessionTopic(String name, String country, String metaData, String comment,
                        String dateAdded, boolean archived, Integer id, Integer addedBy) {
        this.name = name;
        this.country = country;
        this.metaData = metaData;
        this.comment = comment;
        this.dateAdded = dateAdded;
        this.archived = archived;
        this.id = id;
        this.addedBy = addedBy;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getMetaData() {
        return metaData;
    }

    public void setMetaData(String metaData) {
        this.metaData = metaData;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public String getDateAdded() {
        return dateAdded;
    }

    public void setDateAdded(String dateAdded) {
        this.dateAdded = dateAdded;
    }

    public boolean isArchived() {
        return archived;
    }

    public void setArchived(boolean archived) {
        this.archived = archived;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getAddedBy() {
        return addedBy;
    }

    public void setAddedBy(Integer addedBy) {
        this.addedBy = addedBy;
    }
}

File: app/src/main/java/com/expansion/lg/kimaru/training/objs/SessionTopic.kt
package com.expansion.lg.kimaru.training.objs

/**
 * Created by kimaru on 2/21/18.
 */

class SessionTopic {

    var name: String
    var country: String
    var metaData: String
    var comment: String
    var dateAdded: String
    var isArchived: Boolean = false
    var id: Int? = null
    var addedBy: Int? = null

    constructor() {}

    constructor(name: String, country: String, metaData: String, comment: String,
                dateAdded: String, archived: Boolean, id: Int?, addedBy: Int?) {
        this.name = name
        this.country = country
        this.metaData = metaData
        this.comment = comment
        this.dateAdded = dateAdded
        this.isArchived = archived
        this.id = id
        this.addedBy = addedBy
    }
}